geometry_msgs/PoseStamped current
geometry_msgs/PoseStamped target
---
geometry_msgs/PoseStamped best